graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -335,13 +335,12 @@ FusedCSCSamplingGraph::SampleNeighborsImpl(
         auto num_picked_neighbors_data_ptr =
```

Comment:
using indptr_ptr_t = scalar_t;

Reply:
And after that `AT_DISTPATCH_INTEGRAL_TYPES` for nodes as well.

Reply:
We can then write `const int64_t nid = nodes_ptr[i];`. `nodes_ptr` will be a pointer with original dtype of `nodes`.